release/artifacts.yaml

```diff
@@ -45,7 +45,7 @@
 - target: //daml-lf/engine:engine
   type: jar-scala
 - target: //daml-lf/repl:repl
-  type: jar
+  type: jar-scala
```

Comment:
This doesn't need to be released as source, similar to the codegens.

Reply:
👍